croud/clusters/commands.py

```diff
@@ -25,6 +25,18 @@
 from croud.util import require_confirmation
 
 
+def clusters_get(args: Namespace) -> None:
+    """
+    Get cluster by ID
```

Comment:
```suggestion
    Get cluster by ID.
```
as far as i've seen, isn't that the convention ?

Reply:
Makes sense (even we don't follow the convention everywhere).

Reply:
Actually, I don't think the doc strings add much to the commands (anywhere).

Reply:
@turbo-ele Removed them in a new commit.